README.md

```diff
@@ -147,7 +135,7 @@ region_kms_keys = {
 ```
 
 ```console
-AWS_PROFILE=cool-images-ec2amicreate-terraformer-packer packer build --timestamp-ui -var-file release.pkrvars.hcl src/packer.pkr.hcl
+AWS_PROFILE=cool-images-ec2amicreate-terraformer-packer packer build --timestamp-ui -var-file release.pkrvars.hcl packer.pkr.hcl
```

Comment:
Shouldn't `packer.pkr.hcl` be replaced here by `build.pkr.hcl`?

Reply:
I think you're right, and that change should be made upstream.  If @mcdonnnj agrees then I'll create the PR.

Reply:
Actually, the `prerelease.yml` and `release.yml` workflows suggest that `packer.pkr.hcl` should be replaced with `.`.

Reply:
See cisagov/skeleton-packer#384.

Reply:
Ah, right, that makes sense! 👍